src/main/java/organice/logic/parser/AddCommandParser.java

```diff
@@ -84,8 +85,9 @@ public AddCommand parse(String args) throws ParseException {
             Organ organ = ParserUtil.parseOrgan(argMultimap.getValue(PREFIX_ORGAN).get());
             OrganExpiryDate organExpiryDate =
                     ParserUtil.parseOrganExpiryDate(argMultimap.getValue(PREFIX_ORGAN_EXPIRY_DATE).get());
-
-            Donor donor = new Donor(type, nric, name, phone, age, bloodType, tissueType, organ, organExpiryDate);
+            Status status = new Status(Status.STATUS_NOT_PROCESSING);
+            Donor donor = new Donor(type, nric, name, phone, age, bloodType, tissueType, organ, organExpiryDate,
+                    status);
```

Comment:
Since when we add the status always not processing, is it better if we have it in the `Donor` constructor, no need to pass in as parameter so its like `this.status = new Status(Status.STATUS_NOT_PROCESSING)?

Reply:
I dont quite understand this part